plugins/exporter.koplugin/target/joplin.lua

```diff
@@ -305,16 +287,10 @@ function JoplinExporter:getMenuTable()
                 keep_menu_open = true,
                 callback = function()
                     UIManager:show(InfoMessage:new {
-                        text = T(_([[You can enter your auth token on your computer by saving an empty token. Then quit KOReader, edit the exporter.joplin_token field in %1/settings.reader.lua after creating a backup, and restart KOReader once you're done.
-
-To export to Joplin, you must forward the IP and port used by this plugin to the localhost:port on which Joplin is listening. This can be done with socat or a similar program. For example:
-
-For Windows: netsh interface portproxy add v4tov4 listenaddress=0.0.0.0 listenport=41185 connectaddress=localhost connectport=41184
+                        text = T(_([[You can configure markdown formatting by navigating to:
+        Export Highlights > Choose Formats and Services > Markdown
```

Comment:
```suggestion
                        text = T(_([[Markdown formatting can be configured in:
        Export highlights > Choose formats and services > Markdown
```
(if that says Export Highlights and Choose Formats and Services, please remove those capitals there)